### DISCUSSION TITLE
Should we use another format that .story

### DISCUSSION THREAD

**Discussion by elevatebart** [histoire-dev/histoire#6] [Ideas]
Storybook is heavy on branding. And they are great at design too.

If we try to use their wording instead of our own, will it make sense to users?

**Comment by Akryum** [comment]
So "histoires"? :D

Great question though. Maybe we can come up with other wording.

**Reply by elevatebart** [reply]
Ideas off the bat:
- `.hist` shorter but confusing with history being a thing in the Web
- `.examples` more explicit but not on branding
- `.histoires` if we generate the files from a component anyway I like it.

**Reply by hugoattal** [reply]
I think ".histoire" might be a little long and difficult to write on a daily basis for non-french speaking dev 😅...

**Comment by hugoattal** [comment]
I feel like stories may be easier to refer to, as I think that it's becoming a standard word in component-based dev.

But I'm fine with going with something else! Here are a few suggestions:
- `.fiche`
- `.carte`
- `.guide`

**Reply by elevatebart** [reply]
So... I liked the `.doc.vue` because that is what you are writing anyway and there will be a doc bloc in there.

Should we ?

**Reply by hugoattal** [reply]
I'm fine with it!

**Reply by Akryum** [reply]
BTW I feel like stories are also about developing components and visual testing, not just documentation.

**Reply by bartenra** [reply]
`.histoiroy`

In all seriousness, I like `.story`. :laughing:
